src/studio/nuxeo-studio/nuxeo-studio-release-notes/2018-04-17-studio-73-11-1.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'April 17, 2018'
+description: .
+tree_item_index: 988
+review:
+  comment: ''
+  date: '2018-04-17'
+  status: ok
+toc: true
+---
+
+{{! multiexcerpt name='summary-v73-11-1'}}
+This release eases the document type creation process when using Web UI.
+{{! /multiexcerpt}}
+
+## Easier Document Type Creation Process
+When configuring a document type, a new `SVG icon` option is now available if your application has a dependency to the `Web UI` addon.
+
+![]({{file name='svg-icon.png'}} ?border=true,w=445)
+
+From there, you can select any svg file available in the Studio project resources or upload your own. This file will automatically be used in Web UI's document creation popup dialog.
+
+If you already configured your document type icons by uploading the corresponding files in Designer's resources tab, note that they can be kept and will override the configuration made in the SVG icon field.
+
+## Noteworthy Improvements and Bugfixes
+
+- When using Web UI, the resolution actions tab is now visible to configure your workflow tasks buttons  ([NXS-4689](https://jira.nuxeo.com/browse/NXS-4689)).
```

Comment:
When using Web UI, the Resolution Actions tab is now available to configure your workflow tasks buttons